server/utils/courseParticipationUtils.test.ts

```diff
@@ -11,6 +15,68 @@ const getRow = (
 }
 
 describe('CourseParticipationUtils', () => {
+  describe('processedCourseFormData', () => {
+    let request: DeepMocked<Request>
+
+    beforeEach(() => {
+      request = createMock<Request>({})
+    })
+
+    describe('when the `courseId` is a string with length', () => {
+      it('returns the course ID and reports no errors', () => {
+        const validId = 'AN-ID'
+
+        expect(CourseParticipationUtils.processedCourseFormData(validId, undefined, request)).toEqual({
+          courseId: validId,
+          hasFormErrors: false,
+        })
+      })
+    })
+
+    describe("when the `courseId` is `'other'`", () => {
+      const otherId = 'other'
+
+      describe('and `otherCourseName` is a string with length when trimmed', () => {
```

Comment:
maybe "not an empty string when trimmed" would be a bit more human-readable here.

Reply:
I think I lost the will to word here